packages/cli/src/ErrorReporting.ts

```diff
@@ -12,33 +12,59 @@ export const initErrorHandling = async () => {
 	});
 
 	const dsn = config.getEnv('diagnostics.config.sentry.dsn');
-	if (!config.getEnv('diagnostics.enabled') || !dsn) {
+	if (!dsn) {
```

Comment:
Any reason why we are no longer adhering to the diagnostics enabled flag?

Reply:
[this was originally added because I wanted to track errors for all self-hosters](https://github.com/n8n-io/n8n/pull/4394/files#diff-b2b14ce2d590dff81d38e6387ccb790dfdec594c1bfb23cfc347ebdc74154be6R956). That turned out to be very noisy, and I eventually removed the default DSN.
without a default DSN, there is no reason to do this additional check, because if someone is defining a DSN, then they most definitely want error-reporting to work.